python/spacewalk/server/importlib/backend.py

```diff
@@ -913,7 +913,7 @@ def lookupPackageKeyId(self, header):
         sigkeys = rhn_rpm.RPM_Header(header).signatures
         key_id = None  # _key_ids(sigkeys)[0]
         for sig in sigkeys:
-            if sig["signature_type"] == "gpg":
+            if sig["signature_type"] in ["gpg", "pgp", "rsa"]:
```

Comment:
In general no much difference, but if you don't mind, I would prefer to use tuple here instread of the list:
```suggestion
            if sig["signature_type"] in ("gpg", "pgp", "rsa"):
```